File: Freestyle/Webservice/WcfService/Models/DTO/ToDb/UserLogIn.cs
using System.Runtime.Serialization;

namespace WcfService.Models.DTO.ToDb
{
    [DataContract]
    public class UserLogIn
    {
        [DataMember]
        public string Login { get; set; }

        [DataMember]
        public string Pass { get; set; }

        [DataMember]
        public string ClientVer { get; set; }
    }
}

File: Freestyle/Android/Freestyle/src/freestyle/data/dto/todb/UserLogIn.java
package freestyle.data.dto.todb;

public class UserLogIn {
	
	public String login;
	public String pass;
	public String clientVer;
	
	public UserLogIn(String login, String pass, String clientVer){
		this.login = login;
		this.pass = pass;
		this.clientVer = clientVer;
	}
}
